homeassistant/util/package.py

```diff
@@ -60,3 +60,27 @@ def check_package_exists(package: str, lib_dir: str) -> bool:
     # Check packages from global + virtual environment
     # pylint: disable=not-an-iterable
     return any(dist in req for dist in pkg_resources.working_set)
+
+
+def handle_requirements(component: Any, config_dir: str) -> bool:
+    """Install all REQUIREMENTS for the component/dict."""
+    reqs = (getattr(component, 'REQUIREMENTS', None) or
+            (component.get('REQUIREMENTS')
+             if isinstance(component, dict) else None))
+    if reqs is None:
+        return True
+    assert isinstance(reqs, (list, tuple)), ("REQUIREMENTS. List expected, "
+                                             "got {}".format(type(reqs)))
+    deps_dir = mount_lib_path(config_dir)
+    for req in reqs:
+        if not install_package(req, target=deps_dir):
+            raise ImportError('Could not install requirement {}'.format(req))
+    return True
+
+
+def mount_lib_path(config_dir: str) -> str:
```

Comment:
This shouldn't be part of `util`. `util` packages should only contain non-Home Assistant specific code.